server/API.py

```diff
@@ -0,0 +1,10 @@
+import secrets
```

Comment:
The name `API.py` could be ambiguous for whether this is MBTA's v3 api or the api provided by this server to the frontend. Perhaps `V3API.py` or `MBTAAPI.py`?

Reply:
Good point; I'll update it before merge.